.github/workflows/build.yml

```diff
@@ -55,7 +55,14 @@ jobs:
             $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
           sudo apt update
           sudo apt -y install docker-ce docker-ce-cli containerd.io
-          sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+          sudo curl -L "https://github.com/docker/compose/releases/download/v2.11.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+          sudo chmod +x /usr/local/bin/docker-compose
+          sudo ln -s /usr/local/bin/docker-compose /usr/bin/docker-compose
+          docker-compose --version
+      - name: Upgrade docker compose
+        run: |
+          docker-compose --version
+          sudo curl -SL "https://github.com/docker/compose/releases/download/v2.11.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```

Comment:
this line upgrade CI's docker compose from 1.29.2 to 2.11.2, so I can have `--with` in `docker-compose` command line arg, to activate `healthcheck`.

Reply:
ok, now we are moving from `docker-compose` to `docker compose`, removing these lines of manual upgrade to 2.11.2